skycoin/skybian
# add dmsghttp feature

Fixes skywire-services [#423](https://github.com/SkycoinPro/skywire-services/issues/423)

Changes:
- add a checkbox to use dmsghttp configuration on `skyimager`
- download and store dmsghttp-config.json file from skywire repo and put in image on `make build-skybian-img`

How to test:
- clone this branch `git clone git@github.com:mrpalide/skybian.git --branch feature/dsmghttp`
- ~clone skywire@develop branch by `git clone git@github.com:mrpalide/skywire.git --branch develop`~
- ~edit `go.mod` and uncomment `replace github.com/skycoin/skywire => ../skywire`~
- ~`make dep`~
- ~change `DMSGHTTP_CONFIG_URL` on build.conf file to `https://raw.githubusercontent.com/skycoin/skywire/develop/localServers.json`~
- make an skybian image by something like `make build-skybian-img BOARD=rpi64 ARCH=arm64` base on your board. 
- use `make run-skyimager-gui` and check `Use dmsgHTTP`.
- build final image and burn on your SD card.
- after first run of board, check `/etc/skywire-config.json` to see new dmsghttp URLs services.

Does this change need to mentioned in CHANGELOG.md?
Yes
